src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Locale;
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index = new Random().nextInt(Color.values().length);
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[index].toString().toLowerCase(Locale.ROOT);
```

Comment:
don`t use to string for enum

Reply:
you don't need to provide a locale here :)